libraries/botbuilder/src/skills/skillHttpClient.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @module botbuilder
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Activity, ConversationReference, TurnContext } from 'botbuilder-core';
+import { ICredentialProvider } from 'botframework-connector';
+import { InvokeResponse } from '../botFrameworkAdapter';
+import { BotFrameworkHttpClient } from '../botFrameworkHttpClient';
+import { SkillConversationIdFactoryBase } from './skillConversationIdFactoryBase';    
+
+/**
+ * A BotFrameworkHttpClient specialized for Skills that encapsulates Conversation ID generation.
+ */
+export class SkillHttpClient extends BotFrameworkHttpClient {
+    constructor(credentialProvider: ICredentialProvider, private readonly conversationIdFactory: SkillConversationIdFactoryBase, channelService?: string) {
+        super(credentialProvider, channelService);
+    }
+
+    public async postToSkill(fromBotId: string, toSkillId: string, toSkillUrl: string, serviceUrl: string, activity: Activity): Promise<InvokeResponse> {
```

Comment:
It doesn't seem to be using the BotFrameworkSkill class, the goal of this class is to simplify the number of attributes needed to call a skill and also use the factory in a consistent way like the SkillHandler does. 
Are we not porting this 1:1 to match c#?

Reply:
I've changed it to match C# 👍